swap/src/asb/config.rs

```diff
@@ -285,12 +293,25 @@ pub fn query_user_for_initial_config(testnet: bool) -> Result<Config> {
     }
     let ask_spread = Decimal::from_f64(ask_spread).context("Unable to parse spread")?;
 
+    let rendezvous_address = Input::with_theme(&ColorfulTheme::default())
+        .with_prompt("Do you want to advertise your ASB instance with a rendezvous node? Enter an empty string if not.")
+        .default(DEFAULT_RENDEZVOUS_ADDRESS.to_string())
+        .interact_text()?;
+
+    let rendezvous_point = if rendezvous_address.is_empty() {
```

Comment:
Does that make sense? Can it actually be empty?
If it is empty (i.e. the user just hits enter), wouldn't it automatically apply the default value?

Reply:
I never actually tried :joy:

I was hoping that entering `""` would return an empty string and not the default.